sources/dynamodb/schema_test.go

```diff
@@ -269,8 +297,76 @@ func TestProcessSchema_FullDataTypes(t *testing.T) {
 	assert.Equal(t, int64(0), conv.Unexpecteds())
 }
 
+func TestProcessData(t *testing.T) {
+	strA := "str-1"
+	numStr1 := "10.1"
+	numStr2 := "12.34"
+	numVal1 := big.NewRat(101, 10)
+
+	boolVal := true
+	scanOutputs := []dynamodb.ScanOutput{
+		{
+			Items: []map[string]*dynamodb.AttributeValue{
+				{
+					"a": {S: &strA},
+					"b": {N: &numStr1},
+					"c": {N: &numStr2},
+					"d": {BOOL: &boolVal},
```

Comment:
Do you think it makes sense to add more data types here? We do already cover the mappings extensively in the convArray and convScalar unit tests.